packages/builder/src/builder.js

```diff
@@ -525,63 +475,43 @@ export default class Builder {
       }
 
       if (indicatorPath) {
-        templatesFiles.push({
+        templateFiles.push({
           src: indicatorPath,
           dst: 'loading.html',
           custom: customIndicator,
           options: this.options.loadingIndicator
         })
       } else {
-        /* istanbul ignore next */
         // eslint-disable-next-line no-console
```

Comment:
Now do not need to deactivate anymore. :smile::

Reply:
Oh, thank you, sorry for I didn't notice it